### PR TITLE
Add Spacelift Integration

### PR DESCRIPTION
**Pull request by theseanodell**
Add Spacelift Integration to the OPA repository.

Signed-off-by: Sean O'Dell [seano@spacelift.io](mailto:seano@spacelift.io)

### CODE REVIEW

**Comment by philipaconrad** [comment]
Hello @theseanodell! Thanks for putting together this PR. :smile: 

It looks like you'll need to rebase your commits to meet the [DCO](https://github.com/apps/dco) policy. (To include the `Signed-off-by: Author Name ...` bit at the bottom of your commits.) Otherwise, everything else looks good!

File: docs/website/data/integrations.yaml [hunk 1]
```diff
@@ -1067,8 +1067,8 @@ integrations:
       - scalr
     blogs:
       - https://www.scalr.com/blog/opa-is-to-policy-automation-as-terraform-is-to-iac/
-
-  spire:
+      
+    spire:
```
Comment (anderseknert) [inline docs/website/data/integrations.yaml:1071]: This looks like an accidental change.

**Review by anderseknert — COMMENTED** [review]
Except for the unrelated change, looks good to me.

File: docs/website/data/integrations.yaml (hunk 1 above)
Comment (theseanodell) [reply docs/website/data/integrations.yaml]: I have fixed this is a subsequent merge.

**Pull request closed without merging**